indigo_app/static/javascript/indigo/views/work.js

```diff
@@ -149,11 +149,12 @@
     },
 
     canSave: function() {
-      if (this.model.isNew()) {
-        this.$('.btn.save').attr('disabled', !this.dirty || !this.model.isValid() || !this.model.get('publication_date'));
-      } else {
-        this.$('.btn.save').attr('disabled', !this.dirty || !this.model.isValid());
-      }
+      this.$('.btn.save').attr('disabled', !this.dirty || !this.model.isValid());
```

Comment:
The form submission essentially handles this now. Which is better because rather than disabling the button, it provides an error message to the user so they can see what's wrong.